lib/state/actions/pokemon_actions.dart

```diff
@@ -12,3 +12,12 @@ class GetPokemonsAction extends ReduxAction<AppState> {
     return state.copyWith(pokemons: pokemons);
   }
 }
+
+class GetPokemonDetailAction extends ReduxAction<AppState> {
+  @override
+  Future<AppState> reduce() async {
+    final pokeDetail = await ApiService().pokemonApi.getPokemonDetails(name: pokemonNameTest);
```

Comment:
name should be dynamic not fixed value, utilize constructor parameter injection instead 🙂